README.md

```diff
@@ -2,24 +2,250 @@
+    * Note: Individuals may also become Members so long as their direct employer is not a current Member of the ASC. For these purposes, a consultant is not considered a direct employee - thus, someone consulting for a current ASC Member is eligible to become an individual member should they choose to do so. Individual members who subsequently join Member organizations will be absorbed into the larger organization's membership. This likewise applies to individuals who are affiliated with an existing Member organization (e.g., as a contractor), but are not direct employees of that organization. Individuals requesting membership that are affiliated with multiple ASC members can be granted individual Member status by vote of the ASC. This is intended to preclude undue influence in voting procedures by an organization with multiple individual members.
+
+* A Member is removed from the ASC (a) upon having failed to have at least one representative attend two quarterly meetings in the prior twelve months, or (b) by vote of the other ASC members per the voting rules defined below. Attendance by a non-voting member of the organization (i.e., someone not officially designated as an alternate for voting purposes) counts towards meeting this requirement.
+
+The ASC meets quarterly, with at least one quarterly meeting each year to be held "in-person" at an agreed-upon location unless the ASC Members vote otherwise. A vote to "virtualize" the in-person meeting can only apply to that given year. Other quarterly meetings may be held by teleconference in accordance with the desires of ASC members. ASC quarterly meetings are conducted according to the following schedule:
+
+* 3rd week of January
+* 3rd week of April
+* 3rd week of July
+* 3rd week of October
+
+ASC Members may, if desired, vote to alter the timing of the meetings in a given year to accommodate Member schedules, subject to the voting rules described below. Changes to a given meeting's dates must be determined no later than the prior quarterly meeting.
+
+The Co-Chairs shall include links to each proposed modification to the Standard up for consideration during that meeting in the published agenda.
+
+#### Current ASC Members
+
```

Comment:
I don't think current membership belongs in the governance document so that it doesn't have to be versioned each time membership changes. Perhaps the document could refer to a web page or something

Reply:
I believe the governance "doc" is just the readme on the standards repo, so there is no real versioning involved here. I have no strong feelings about it, but the doc does say that members are given visible acknowledgement of their participation, and this seems a relatively simple place to provide it.